src/NLog.Targets.Seq/Levels.cs

```diff
@@ -0,0 +1,101 @@
+                ["tr"] = ("Trace", LogLevel.Trace),
+                ["trc"] = ("Trace", LogLevel.Trace),
+                ["trce"] = ("Trace", LogLevel.Trace),
+                ["trace"] = ("Trace", LogLevel.Trace),
+                ["v"] = ("Verbose", LogLevel.Trace),
+                ["ver"] = ("Verbose", LogLevel.Trace),
+                ["vrb"] = ("Verbose", LogLevel.Trace),
+                ["verb"] = ("Verbose", LogLevel.Trace),
+                ["verbose"] = ("Verbose", LogLevel.Trace),
+                ["d"] = ("Debug", LogLevel.Debug),
+                ["de"] = ("Debug", LogLevel.Debug),
+                ["dbg"] = ("Debug", LogLevel.Debug),
+                ["deb"] = ("Debug", LogLevel.Debug),
+                ["dbug"] = ("Debug", LogLevel.Debug),
+                ["debu"] = ("Debug", LogLevel.Debug),
+                ["debug"] = ("Debug", LogLevel.Debug),
+                ["i"] = ("Information", LogLevel.Info),
+                ["in"] = ("Information", LogLevel.Info),
+                ["inf"] = ("Information", LogLevel.Info),
+                ["info"] = ("Information", LogLevel.Info),
+                ["information"] = ("Information", LogLevel.Info),
+                ["notice"] = ("Notice", LogLevel.Info),
+                ["w"] = ("Warning", LogLevel.Warn),
+                ["wa"] = ("Warning", LogLevel.Warn),
+                ["war"] = ("Warning", LogLevel.Warn),
+                ["wrn"] = ("Warning", LogLevel.Warn),
+                ["warn"] = ("Warning", LogLevel.Warn),
+                ["warning"] = ("Warning", LogLevel.Warn),
+                ["e"] = ("Error", LogLevel.Error),
+                ["er"] = ("Error", LogLevel.Error),
+                ["err"] = ("Error", LogLevel.Error),
+                ["erro"] = ("Error", LogLevel.Error),
+                ["eror"] = ("Error", LogLevel.Error),
+                ["error"] = ("Error", LogLevel.Error),
+                ["f"] = ("Fatal", LogLevel.Fatal),
+                ["fa"] = ("Fatal", LogLevel.Fatal),
+                ["ftl"] = ("Fatal", LogLevel.Fatal),
+                ["fat"] = ("Fatal", LogLevel.Fatal),
+                ["fatl"] = ("Fatal", LogLevel.Fatal),
+                ["fatal"] = ("Fatal", LogLevel.Fatal),
+                ["c"] = ("Critical", LogLevel.Fatal),
+                ["cr"] = ("Critical", LogLevel.Fatal),
+                ["crt"] = ("Critical", LogLevel.Fatal),
+                ["cri"] = ("Critical", LogLevel.Fatal),
+                ["crit"] = ("Critical", LogLevel.Fatal),
+                ["critical"] = ("Critical", LogLevel.Fatal),
+                ["emerg"] = ("Emergency", LogLevel.Fatal),
+                ["panic"] = ("Panic", LogLevel.Fatal)
+            };
+
+        const string LevelMarker = "\"MinimumLevelAccepted\":\"";
+
+        /// <summary>Maps a level to an NLog equivalent level.</summary>
+        /// <returns>NLog level; gauranteed not null</returns>
+        public static LogLevel ToNLogLevel(this string level)
```

Comment:
Nit: since `string` is a widely-used built-in type that doesn't always make sense to treat as a level, it may be nicer to use a non-extension method, here.

Reply:
It's true that someone could potentially use that on a string that's not a level, but there's no difference between that and using `string.ToUpper()` on a string that's likely to contain a telephone number. So I think this really comes down to a stylistic choice and in the context it is used, I felt that it helped readability. But then I'm a fan of fluent interfaces and perhaps my preference is unusual. One of the problems of being a lone-developer is that you sometimes develop idiosyncratic approaches to things ;-)

Reply:
OK I have converted those to non-extension methods. It hurts my aesthetic sensibilities to do that but its not my codebase so I think this is the right thing to do.